substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/BundleSupport.java

```diff
@@ -182,6 +313,192 @@ static BundleSupport create(NativeImage nativeImage, String bundleArg, NativeIma
         }
     }
 
+    private void initializeContainerImage() {
+        String bundleFileName = bundlePath == null ? "" : bundlePath.resolve(bundleName + BUNDLE_FILE_EXTENSION).toString();
+
+        if(bundleDockerfile != null && dockerfile == null) {
+            dockerfile = bundleDockerfile;
+        }
+
+        // create Dockerfile if not available for writing or loading bundle
+        try {
+            if (dockerfile == null) {
+                dockerfile = Files.createTempFile("Dockerfile", null);
+                String dockerfileText = DEFAULT_DOCKERFILE;
+                if(nativeImage.getNativeImageArgs().contains("--static") && nativeImage.getNativeImageArgs().contains("--libc=musl")) {
+                    dockerfileText += System.lineSeparator() + DEFAULT_DOCKERFILE_MUSLIB;
+                }
+                Files.write(dockerfile, dockerfileText.getBytes());
```

Comment:
Use `java.nio.file.Files#writeString`

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/31d09e9077643d268248503201bfc730860b72a0